gnmi_server/server.go

```diff
@@ -140,11 +143,11 @@ func NewServer(config *Config, opts []grpc.ServerOption) (*Server, error) {
 	}
 	gnmipb.RegisterGNMIServer(srv.s, srv)
 	spb_jwt_gnoi.RegisterSonicJwtServiceServer(srv.s, srv)
-	if READ_WRITE_MODE {
+	if srv.config.MgmtEnable {
```

Comment:
>MgmtEnable [](http://example.com/codeflow?start=15&length=10)

Suggest only refactor in this PR:
1. keep original naming: `srv.config.ReadWriteMode`
2. I guess your new feature is `MixedEnable`. Let's add it later in another PR.